pyproject.toml

```diff
@@ -74,3 +74,45 @@ profile = "True"
 # the end when publishing non-tagged versions.
 [tool.setuptools_scm]
 write_to = "src/pylibsshext/_scm_version.py"
+
+[tool.towncrier]
+package = "tox"
+filename = "docs/changelog.rst"
+directory = "docs/changelog/fragments"
+template = "docs/changelog/template.jinja2"
+title_format = "v{version} ({project_date})"
+issue_format = "`#{issue} <https://github.com/tox-dev/tox/issues/{issue}>`_"
+underlines = ["-", "^"]
+
+[[tool.towncrier.section]]
+    path = ""
+
+[[tool.towncrier.type]]
+    directory = "bugfix"
+    name = "Bugfixes"
+    showcontent = true
+
+[[tool.towncrier.type]]
+    directory = "feature"
+    name = "Features"
+    showcontent = true
+
+[[tool.towncrier.type]]
+    directory = "deprecation"
+    name = "Deprecations (removal in next major release)"
+    showcontent = true
+
+[[tool.towncrier.type]]
+    directory = "breaking"
+    name = "Backward incompatible changes"
+    showcontent = true
+
+[[tool.towncrier.type]]
+    directory = "doc"
+    name = "Documentation"
+    showcontent = true
+
+[[tool.towncrier.type]]
+    directory = "misc"
+    name = "Miscellaneous"
+    showcontent = true
```

Comment:
This breaks the assumption of having another section the last. See the comment above, it should be relocated to keep the scm setup last